### PR TITLE
remove withdrawn req

### PR DESCRIPTION
**Pull request by RoriCremer**
Allow CH to create custom cohorts with a subset of samples

Tested with sample_id=1 (sample name = ERS4367795)

Imported:
Screen Shot 2022-05-13 at 4 13 15 PM

In the alt-allele
Screen Shot 2022-05-17 at 6 52 30 PM

In the filter:
Screen Shot 2022-05-17 at 6 51 43 PM

In the prepare:
Screen Shot 2022-05-17 at 2 47 32 PM

In the extract:
Screen Shot 2022-05-17 at 6 48 17 PM

### CODE REVIEW

**Comment by kcibul** [comment]
As part of this, do we want to also remove this restriction in the other places we put it in?   A quick grep turns up

GvsExtractCallset.wdl
GvsCreateFilterSet.wdl
populate_alt_allele_table.py
GvsImportGenomes.wdl

**Comment by RoriCremer** [comment]
Bec and I were just having this discussion---do we want to just unblock CH asap or do we want to remove this everywhere---but given that we haven't unblocked him yet and seem to be blocked on the date / timestamp---yes, let's remove them all!

File: scripts/variantstore/wdl/GvsExtractCallset.wdl [hunk 1]
```diff
@@ -388,11 +388,6 @@ task CreateManifest {
 }
 
 task GenerateSampleListFile {
-  # should not call cache in case of withdrawn samples
-  meta {
-    volatile: true
-  }
-
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:395]: do we want to retain this?

Comment (gbggrant) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl:395]: I think you should. It's entirely dependent on the database state.

**Review by gbggrant — COMMENTED** [review]
I think the volatile = true *should* be retained.

File: scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py [hunk 2]
```diff
@@ -89,7 +89,7 @@ def get_all_sample_ids(fq_destination_table_samples):
 def create_extract_samples_table(control_samples, fq_destination_table_samples, fq_sample_name_table, fq_sample_mapping_table):
   sql = f"CREATE OR REPLACE TABLE `{fq_destination_table_samples}` AS (" \
         f"SELECT m.sample_id, m.sample_name, m.is_loaded, m.withdrawn, m.is_control FROM `{fq_sample_name_table}` s JOIN `{fq_sample_mapping_table}` m ON (s.sample_name = m.sample_name) " \
-        f"WHERE m.is_loaded is TRUE AND m.withdrawn IS NULL AND m.is_control = {control_samples})"
+        f"WHERE m.is_loaded is TRUE AND m.is_control = {control_samples})"
```
Comment (kcibul) [inline scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py:92]: separate discussion, but I wonder if this encoding of "control" samples is another case where we put biz logic into GVS?

**Review by kcibul — APPROVED** [review]

File: scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py (hunk 2 above)
Comment (RoriCremer) [reply scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py:92]: maybe---but the biz logic is OURS for sensitivity and precision

File: scripts/variantstore/wdl/GvsExtractCallset.wdl (hunk 1 above)
Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl]: ok---no longer removed as part of the pr!

**Comment by RoriCremer** [comment]
other than manually testing this---is there a test I can write to include this? Maybe add it to the QS integration test?

**Pull request merged**